src/styles/blocks/container.scss

```diff
@@ -0,0 +1,40 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  max-width: 1199px;
```

Comment:
What is it? Your boxes do not look stretched on the widescreen.
```suggestion
```

Reply:
When 3 blocks show up it is stretching from 900px till 1199px and then does not grow any longer. And all tests have passed so what's wrong with that?